apps/webapp/src/state/ibc.ts

```diff
@@ -57,32 +62,37 @@ export const createIbcSendSlice = (): SliceCreator<IbcSendSlice> => (set, get) =
         state.ibc.destinationChainAddress = addr;
       });
     },
-    sendIbcWithdraw: async toastFn => {
+    sendIbcWithdraw: async () => {
       set(state => {
         state.send.txInProgress = true;
       });
 
-      const { dismiss, update } = toastFn(buildingTxToast());
+      const txToastId = toast(...buildingTxToast());
 
       try {
         const transactionPlan = await plan(await getPlanRequest(get().ibc));
-        const transaction = await authWitnessBuild(update, { transactionPlan });
-        const { txHash, detectionHeight } = await broadcast(update, { transaction });
+        const transaction = await authWitnessBuild({ transactionPlan }, status =>
+          toast(...buildingTxToast(status, txToastId)),
+        );
+        const txHash = await getTxHash(transaction);
+        const { detectionHeight } = await broadcast({ transaction }, status =>
+          toast(...broadcastingTxToast(txHash, status, txToastId)),
+        );
 
-        update(successTxToast(txHash, detectionHeight));
+        toast(...successTxToast(txHash, detectionHeight));
```

Comment:
_(referencing https://sonner.emilkowal.ski/)_

Could we make use of the different toast "types"? Looks like success/error/promise could be useful here?

Can we also use the `richColors` setting?

Reply:
ah, yeah, I did that in a few places but missed this, apparently. I'll fix it (which will be easier when using your approach above re: calling `toast()` from the `toast-content` helpers).

Reply:
<img width="431" alt="image" src="https://github.com/penumbra-zone/web/assets/1121544/677224de-ddbe-46e1-9abb-f867520810be">

done